base-dockerfiles/GradleDockerfile

```diff
@@ -0,0 +1,9 @@
+FROM eclipse-temurin:17.0.5_8-jdk-jammy as builder
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY ./gradle/ /app/gradle/
```

Comment:
man könnte hier ausnutzen das man ja schon in app ist also nach ./gradle kopieren

Reply:
ausgenutzt.